astropy/utils/xml/writer.py

```diff
@@ -159,14 +160,13 @@ def xml_cleaning_method(self, method="escape_xml", **clean_kwargs):
 
         if method == "bleach_clean":
             # NOTE: bleach is imported locally to avoid importing it when
-            # it is not nocessary
-            try:
-                import bleach
-            except ImportError:
+            # it is not necessary
+            if not HAS_BLEACH:
                 raise ValueError(
```

Comment:
Wonder if we should make this `ModuleNotFound` too...  Though fine not to in this PR.

Reply:
Actually there are a couple other places where arbitrary exception types are used in place of `ImportError/ModuleNotFoundError`, but since changing types (for a non-subtype of the original) is a subtle API change, I'd prefer addressing them in a separate smaller PR, if that's alright with you.

Reply:
Yes, fine to not do here.